src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -122,6 +124,13 @@ function initStore() {
                         }
 
                         updateStoreFromQuery(resultsViewPageStore, query, samplesSpecification, cancerStudyIds, oql, cancerStudyIds);
+
+                        onMobxPromise(resultsViewPageStore.studyIds, ()=>{
```

Comment:
when there are virtual studies, we want to track the physical studies, so we need to wait for resolution of studyIds